lighthouse-core/test/report/html/renderer/report-ui-features-test.js

```diff
@@ -70,6 +70,8 @@ describe('ReportUIFeatures', () => {
       };
     };
 
+    global.window.location = 'http://localhost';
```

Comment:
Is this necessary? Things got a little confusing in here with recent updates.

@wardpeet Do you know which `window` we should be using? Especially when looking at how this will merge into the updated version on master https://github.com/GoogleChrome/lighthouse/blob/33ba60cc2baa8704cb9dab92dab88407c25fdcb7/lighthouse-core/test/report/html/renderer/report-ui-features-test.js#L55-L73

Should we be creating our own `window` anymore or just using `document.window` (or `documentReport.window`, both of which provide a `location` for us)?

Reply:
> global.window.location = 'http://localhost';

I think we still need to fix this, but it's a bigger fix with `window` in the jsdom tests, so can wait for a future PR.